packages/gatsby-theme-newrelic/README.md

```diff
@@ -9,100 +9,101 @@ websites](https://opensource.newrelic.com).
-- [Components](#components)
-  - [`Banner`](#banner)
-  - [`Button`](#button)
-  - [`Callout`](#callout)
-  - [`CodeBlock`](#codeblock)
-  - [`Collapser`](#collapser)
-  - [`CollapserGroup`](#collapsergroup)
-  - [`ContributingGuidelines`](#contributingguidelines)
-  - [`CookieConsentDialog`](#cookieconsentdialog)
-  - [`CreateIssueButton`](#createissuebutton)
-  - [`Dropdown`](#dropdown)
-    - [`Dropdown.Toggle`](#dropdowntoggle)
-    - [`Dropdown.Menu`](#dropdownmenu)
-    - [`Dropdown.MenuItem`](#dropdownmenuitem)
-  - [`ExternalLink`](#externallink)
-  - [`EditPageButton`](#editpagebutton)
-  - [`FeatherSVG`](#feathersvg)
-  - [`GitHubIssueButton`](#githubissuebutton)
-    - [Environment information](#environment-information)
-  - [`GlobalFooter`](#globalfooter)
-  - [`GlobalHeader`](#globalheader)
-  - [`HamburgerMenu`](#hamburgermenu)
-  - [`Icon`](#icon)
-    - [Available icons](#available-icons)
-    - [Shadowing icons](#shadowing-icons)
-  - [`Layout`](#layout-1)
-    - [`Layout.Content`](#layoutcontent)
-    - [`Layout.Footer`](#layoutfooter)
-    - [`Layout.Main`](#layoutmain)
-    - [`Layout.PageTools`](#layoutpagetools)
-    - [`Layout.Sidebar`](#layoutsidebar)
-  - [`Link`](#link)
-  - [`MarkdownContainer`](#markdowncontainer)
-  - [`MDX`](#mdx)
-    - [Using `MDX`](#using-mdx)
-    - [Default components](#default-components)
-  - [`MDXCodeBlock`](#mdxcodeblock)
-  - [`Navigation`](#navigation)
-  - [`NavItem`](#navitem)
-    - [`Page`](#page)
-  - [`NewRelicLogo`](#newreliclogo)
-  - [`PageTools`](#pagetools)
-    - [`PageTools.Section`](#pagetoolssection)
-    - [`PageTools.Title`](#pagetoolstitle)
-  - [`RelatedResources`](#relatedresources-1)
-  - [`SearchInput`](#searchinput)
-  - [`SEO`](#seo)
-  - [`SimpleFeedback`](#simplefeedback)
-  - [`Spinner`](#spinner)
-  - [`Surface`](#surface)
-  - [`Table`](#table)
-  - [`TableOfContents`](#tableofcontents)
-  - [`Tag`](#tag)
-  - [`TagList`](#taglist)
-  - [`Terminal`](#terminal)
-  - [`TextHighlight`](#texthighlight)
-  - [`Video`](#video)
-- [MDX Component variants](#mdx-component-variants)
-- [Hooks](#hooks)
-  - [`useActiveHash`](#useactivehash)
-  - [`useClipboard`](#useclipboard)
-  - [`useFormattedCode`](#useformattedcode)
-  - [`useInstrumentedData`](#useinstrumenteddata)
-  - [`useInstrumentedHandler`](#useinstrumentedhandler)
-  - [`useKeyPress`](#usekeypress)
-  - [`useLayout`](#uselayout)
-  - [`useQueryParams`](#usequeryparams)
-  - [`useTessen`](#usetessen)
-  - [`useTimeout`](#usetimeout)
-  - [`useUserId`](#useuserid)
-  - [`usePrevious`](#useprevious)
-- [I18n](#i18n-1)
-- [Announcements](#announcements)
-- [Utils](#utils)
-  - [`formatCode`](#formatcode)
-- [Testing](#testing)
-  - [Changes to Jest config](#changes-to-jest-config)
-  - [Mocking `ReactDOM.createPortal` for snapshot testing](#mocking-reactdomcreateportal-for-snapshot-testing)
+- [`gatsby-theme-newrelic`](#gatsby-theme-newrelic)
```

Comment:
Do we need to indent the _entire list_ for a single bullet point? The more indentation we add to this list, the more error prone it becomes ("am I missing a space?")

Reply:
I use a Markdown linter and  a MD table generator, so it's all in the best practice of markdown. It auto corrects on save of a file, but I can modify this if you feel it's an issue.

Reply:
I don't feel strongly about it, we can leave it in.

That said, this isn't best practice because there isn't a best practice with markdown lists. Your linter formats them this way because your plugin's author has an opinion on how this should look. The rest of the team _doesn't_ have that plugin, which is why it _isn't_ currently indented.